gradoop-flink/src/test/java/org/gradoop/flink/algorithms/gelly/hits/HITSTest.java

```diff
@@ -0,0 +1,27 @@
+package org.gradoop.flink.algorithms.gelly.hits;
+
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.junit.Test;
+
+public class HITSTest extends GradoopFlinkTestBase {
+
+  @Test
+  public void minimalHITSTest() throws Exception {
+
+    String inputString = "input[" + "(v0:v0) -[:e]-> (v2:v2) " + "(v1:v1) -[:e]-> (v2) " + "]";
```

Comment:
I think the extra space around the edges is not necessary (didn't even know that GDL accepts it :)).  And maybe use "better" labels for clarity. Also the String concatenation is not necessary here. You can separate pattern with comma or just link them to a path, e.g.:
`String inputString = "input[(v0:A)-[:e]->(v1:B)<[:e]-(v2:A)]";`